src/UploadCommand.php

```diff
@@ -27,10 +27,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->loadConfig($input);
 
         $filename = $input->getArgument('filename');
-        $xml = file_get_contents($filename);
+        $XMLPhonebookStr = file_get_contents($filename);
+        $XMLPhonebook = simplexml_load_string($XMLPhonebookStr);
```

Comment:
maybe `xmlStr` and `xml`

Reply:
if you stick to your opinion after my above explanation, I'll go for your comment.
It's your repo.